examples/validate_relative_binding.py

```diff
@@ -333,81 +327,85 @@ def predict_dG(blocker, mol):
         )
         complex_decouple_x0 = np.concatenate([complex_decouple_x0, aligned_mol_coords, ref_coords])
 
-        (
-            complex_decouple_params,
-            dG_complex_decouple_model,
-            dG_complex_decouple_futures,
-        ) = binding_model_complex_decouple.simulate_futures(
-            ordered_params,
-            mol,
-            blocker_mol,
-            core_idxs,
-            complex_decouple_x0,
-            complex_box0,
-            prefix="complex_decouple_" + mol_name + "_" + str(epoch),
-        )
-
         # compute the free energy of conversion in complex
         complex_conversion_x0 = minimizer.minimize_host_4d(
-            [mol], complex_system, complex_host_coords, forcefield, complex_box0, [aligned_mol_coords]
-        )
-        complex_conversion_x0 = np.concatenate([complex_conversion_x0, aligned_mol_coords])
-        (
-            complex_conversion_params,
-            complex_conversion_model,
-            complex_conversion_futures,
-        ) = binding_model_complex_conversion.simulate_futures(
-            ordered_params,
-            mol,
-            complex_conversion_x0,
+            [mol],
+            complex_system,
+            complex_host_coords,
+            forcefield,
             complex_box0,
-            prefix="complex_conversion_" + mol_name + "_" + str(epoch),
+            [aligned_mol_coords],
         )
+        complex_conversion_x0 = np.concatenate([complex_conversion_x0, aligned_mol_coords])
 
-        # solvent
         min_solvent_coords = minimizer.minimize_host_4d([mol], solvent_system, solvent_coords, forcefield, solvent_box)
         solvent_x0 = np.concatenate([min_solvent_coords, mol_coords])
         solvent_box0 = solvent_box
-        (
-            solvent_conversion_params,
-            solvent_conversion_model,
-            solvent_conversion_futures,
-        ) = binding_model_solvent_conversion.simulate_futures(
-            ordered_params, mol, solvent_x0, solvent_box0, prefix="solvent_conversion_" + mol_name + "_" + str(epoch)
-        )
-        (
-            solvent_decouple_params,
-            solvent_decouple_model,
-            solvent_decouple_futures,
-        ) = binding_model_solvent_decouple.simulate_futures(
-            ordered_params, mol, solvent_x0, solvent_box0, prefix="solvent_decouple_" + mol_name + "_" + str(epoch)
-        )
 
+        suffix = f"{mol_name}_{epoch}"
+
+        return {
```

Comment:
No strong preference but I've seen @maxentile use a NamedTuple for these types of things. Do we have a preference?

Reply:
I would like to encapsulate all of these models into a larger RABFE model, as there is a lot of code in this script that seems specific to setting up and running RABFE simulations. This sort of thing should be a lot more hidden than it is.

Reply:
Makes sense to me - thanks.